internal/host/hostcommands/instruction_manager.go

```diff
@@ -130,6 +139,10 @@ func NewInstructionManager(log logrus.FieldLogger, db *gorm.DB, hwValidator hard
 			models.HostStatusUnbinding:                  {[]CommandGetter{noopCmd}, 0, models.StepsPostStepActionExit},
 			models.HostStatusUnbindingPendingUserAction: {[]CommandGetter{noopCmd}, 0, models.StepsPostStepActionExit},
 		},
+		reclaimHostToSteps: stateToStepsMap{
```

Comment:
Why add another map? Can these just be in `poolHostToSteps`?

Reply:
Modified it to be under the poolHostToSteps, thank you!